gulpfile.js

```diff
@@ -476,7 +476,10 @@ function getHash(cb) {
 
 function writeChangesetId() {
     var versionJson = new stream.Readable;
-    versionJson.push(JSON.stringify({ gitChangesetId: gitChangeSetId }, undefined, 2));
+    versionJson.push(JSON.stringify({
+        gitChangesetId: gitChangeSetId,
+        version: pkg.version
```

Comment:
We should probably respect `version_name` in here as well, like we did in `getManifestVersion()`, in case we want to do RC versions again at some point.